tools/update-rc3.in

```diff
@@ -982,6 +979,8 @@ download_osimage() (
 
     [ -s "${aTGT_FILE}" ] && [ -s "${aTGT_FILE_CSOLD}" ] && [ -s "${aTGT_FILE_MANIFEST}" ] && \
     touch -r "${aTGT_FILE}" "${aTGT_FILE_CSOLD}" "${aTGT_FILE_MANIFEST}" && \
+    mkdir -p /var/lib/fty/sw-update/ && \
```

Comment:
For master and backport: any chown to do here? Or would an ipc-meta-setup snippet take care of this carefully? or both?